plugins/Morpheus/templates/ajaxMacros.twig

```diff
@@ -38,4 +37,23 @@
             {%- endif %}.
         </div>
     </div>
+    <div id="loadingRateLimitError" style="display: none">
+        <div class="alert alert-danger">
+            {{ 'General_ErrorRateLimit'|translate }}
+
+            <br /><br />
+            {{ 'General_NeedMoreHelp'|translate }}
+
+            {% if showMoreHelp %}
+                <a rel="noreferrer noopener" target="_blank" href="https://matomo.org/faq/troubleshooting/faq_19489/">{{ 'General_Faq'|translate }}</a> –
+            {% endif %}
```

Comment:
Is this FAQ link useful to show when we know the error is a 429? The FAQ is quite generic and contains a lot of instructions that aren't going to help if rate limiting is the issue. Maybe if we can show a link to the API documentation then the FAQ link can be omitted for 429 errors?